### DISCUSSION TITLE
Running on multiple GPUs

### DISCUSSION THREAD

**Discussion by Crubert-digit** [texturedesign/texturize#41] [Ideas]
Hi!
First, Thanks for texturize, it's a really great job.

I was wondering if it's possible to run the application on multiple GPUs. I'm trying to mashup on images of size 1024x1024px and it takes a long time depending on the parameters(quality, VGGXX, octaves, etc.), between 5 and 30 minutes. 

My system has two Nvidia GPUs of 16GB but only one is being used during the process. So now I'm trying to compute the mashup on both GPU simultaneously to reduce the time.

**Comment by alexjc** [comment]
I haven't worked on multi-GPU for this code yet.  If you're willing to sacrifice some quality, then splitting up the image and running multiple critics in parallel on different tiles could do the trick...  I'm not sure how the current code would need to be changed, but I'll keep it in mind for the next iteration!
